src/actions/membershipActions.js

```diff
@@ -1,19 +1,21 @@
 import * as types from './actionTypes';
 import membershipApi from '../api/membershipApi';
 import { loadProjects } from './projectActions';
-// TODO: Fix API to return only uniq memberships
-import { uniqBy } from 'lodash';
+import { uniqBy, filter } from 'lodash';
 
-// TODO: Fix API to return only uniq memberships
-const getUniqMemberships = memberships => uniqBy(memberships, membership => membership.user_email);
+const filterMemberships = memberships => filter(memberships, membership =>
+  membership.project_name !== 'Internals'
+);
+
+const getUniqMemberships = memberships =>
+  uniqBy(filterMemberships(memberships), membership => membership.user_email);
```

Comment:
I would not include the filterMemberships function inside the getUniqMemberships because the two functions do different things. To have a better reusability you can call getUniqMembers(filterMemberships(memberships))